contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -23,30 +23,6 @@ $BODY$
 $BODY$
 LANGUAGE SQL STABLE;
 
-CREATE OR REPLACE FUNCTION sys.get_min_id_from_table(IN id_colname TEXT,
```

Comment:
Let's add upgrade script changes as well, e.g.,
- rename it to deprecated ones
- try to drop functions

Reply:
I'll rename it to deprecated ones.